PyPI build and upload workflow crashes at upload step

During the recent release of ``v0.14.0`` the GHA workflow for building and uploading the package to the PyPI crashed. It seems that the build was successful but the workflow crashed during the login into PyPI. 

The error callback: 
```
Uploading distributions to https://upload.pypi.org/legacy/
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/runner/.local/lib/python3.8/site-packages/twine/__main__.py", line 51, in <module>
    sys.exit(main())
  File "/home/runner/.local/lib/python3.8/site-packages/twine/__main__.py", line 33, in main
    error = cli.dispatch(sys.argv[1:])
  File "/home/runner/.local/lib/python3.8/site-packages/twine/cli.py", line 124, in dispatch
    return main(args.args)
  File "/home/runner/.local/lib/python3.8/site-packages/twine/commands/upload.py", line 198, in main
    return upload(upload_settings, parsed_args.dists)
  File "/home/runner/.local/lib/python3.8/site-packages/twine/commands/upload.py", line 127, in upload
    repository = upload_settings.create_repository()
  File "/home/runner/.local/lib/python3.8/site-packages/twine/settings.py", line 329, in create_repository
    self.username,
  File "/home/runner/.local/lib/python3.8/site-packages/twine/settings.py", line 131, in username
    return cast(Optional[str], self.auth.username)
  File "/home/runner/.local/lib/python3.8/site-packages/twine/auth.py", line 34, in username
    return utils.get_userpass_value(
  File "/home/runner/.local/lib/python3.8/site-packages/twine/utils.py", line 248, in get_userpass_value
    value = prompt_strategy()
  File "/home/runner/.local/lib/python3.8/site-packages/twine/auth.py", line 85, in username_from_keyring_or_prompt
    return self.prompt("username", input)
  File "/home/runner/.local/lib/python3.8/site-packages/twine/auth.py", line 96, in prompt
    return how(f"Enter your {what}: ")
EOFError: EOF when reading a line
Enter your username: 
Error: Process completed with exit code 1.
```

--- a/.github/workflows/package_upload.yml
+++ b/.github/workflows/package_upload.yml
@@ -20,10 +20,9 @@ jobs:
           python -m pip install setuptools --user
           python -m pip install build --user
           python -m pip install twine --user
-    - name: PyPI - Build and publish
+    - name: Build distribution
       run: |
           python setup.py sdist
-          python -m twine upload dist/*
     - name: Publish distribution to PyPI
       uses: pypa/gh-action-pypi-publish@master
       with:

